backend/czi_hosted/common/config/dataset_config.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, tag, app_config, default_config):
             raise ConfigurationError(f"Unexpected config: {str(e)}")
 
         # The annotation object is created during complete_config and stored here.
-        self.user_annotations = None
+        self.user_annotations = Annotations()
```

Comment:
Why is this being initialized here and in complete_config? Should the comment above it be updated?

Reply:
The "old" Annotations class was an ABC, as it had a hard requirement on configuration.  The new class is _not_ abstract, as it can provide read of gene sets without config (it can derive their location from the dataset location).   So the default setup, barring additional config, is to use the default behavior.

The comment is a bit misleading, as the `complete_config` may create a different Annotation instance, and replace the default one.  This happens, for example, if the user has enabled the "local csv" annotation class.

I'll update the comment.